### PR TITLE
Course Roadmap - Start Here

### PR DESCRIPTION
**Pull request by sccosel**
I thought this roadmap would be easier to find if it was its own .md file on the front page along with the README.md file.

### CODE REVIEW

**Comment by sccosel** [comment]
Permanent replacement for #53.

**Comment by binary-sequence** [comment]
I would put this content in the README.md

**Comment by MikeBauerCA** [comment]
In the README, there are links to the slack channel and to the registration site for the environment to complete the lab. If the registration process for the system is unclear, perhaps the registration page should be updated.

There are a ton of links in this PR. Thanks for gathering these all up! We should review these links and ensure we are directing folks to appropriate and relevant content. If we are going to have a lot of links, I think we should separate them out into an "additional resources" document and reference it in the README. Also, I would recommend just placing the most relevant content first. I don't see a need to separate resources by vendor.

**Comment by MikeBauerCA** [comment]
We should also try to place the content for folks consuming the course at the very top of the README and leave the build, contributing, and governance sections for later. I'm thinking some of this content could replace what is currently in the "How to use" section of the README and that this PR will help address issues like #28.

**Comment by paulnewt** [comment]
I had a similar discussion with Sudharsana about this yesterday - a review of the README is coming for a number of reasons - but the needed governance changes should be secondary to target audience - those that are looking to learn COBOL

File: Course Roadmap - Start Here.md [hunk 1]
```diff
@@ -24,8 +24,10 @@ FAQ:
 
 **IBM Resources:**
 
-**COBOL Fridays:** 
-[A Beginner's Practical Approach to COBOL](https://developer.ibm.com/technologies/cobol/videos/cobol-programming-with-vscode-course-launch-webinar) - Webinar - Originally broadcast: 2020-04-17 - 1 hour 
+**COBOL Fridays:**
+
+[A Beginner's Practical Approach to COBOL](https://developer.ibm.com/technologies/cobol/videos/cobol-programming-with-vscode-course-launch-webinar) - Webinar - Originally broadcast: 2020-04-17 - 1 hour
+
 [Get hands on! Run a COBOL program](https://developer.ibm.com/technologies/cobol/videos/get-hands-on-run-a-cobol-program) - Webinar - Originally broadcast: 2020-04-24 - 45 minutes
```
Comment (peraciodias) [inline Course Roadmap - Start Here.md:34]: fiz um programa cobol no mainframe mas estou com com problemas !
1- ler arquivo texto com 80 caracteres 
2-criar  novos dados em seguida criar um novo membro com mais de 160 caracteres por registro.
sempre vejo erro de imcompatibilidade na abertura do arquivo !  não sei como fazer , alguém sabe ?

File: Course Roadmap - Start Here.md [hunk 2]
```diff
@@ -24,8 +24,10 @@ FAQ:
...
 
 Next Cobol webinar: Fri, May 1, 2020, 11:00-12:00 Noon EDT:
```
Comment (peraciodias) [inline Course Roadmap - Start Here.md:36]: I made a cobol program on the mainframe but I'm in trouble!
1 - read text file with 80 characters
2 - create new data then create a new member with more than 160 characters per record.
I always see error

**Comment by MikeBauerCA** [comment]
Per our TSC discussion, we are closing this PR and plan to add an FAQ document as well as a document with a list of vetted links to the repository. In the interim, folks can continue to refer to issue #53 for some of this information.

**Pull request closed without merging**